pset-iterators.js

```diff
@@ -129,6 +150,14 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const livingPresidents = presidents.filter(stayinAlive => {
```

Comment:
🛠 Read the comment below and update your code accordingly

@Defy1x fix the indentation issue with `if` statement

Reply:
Done